assembl/static2/js/app/utils/translations.js

```diff
@@ -5,6 +5,7 @@ const Translations = {
     here: "ici",
     yes: "oui",
     no: "non",
+    and: " et ",
```

Comment:
I think you can handle spaces with CSS instead (:before, :after)

Reply:
problem is it's only in the case where we want an 'et' dans we want those spaces :) so we would have to put conditional classes which is a bit overkill for the rendering of those spaces